src/pmdas/openvswitch/switch.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+sudo ovs-vsctl --format=json list bridge 
```

Comment:
This script needs to be removed, ovs-vsctl (and options) should be invoked directly by the PMDA.
As a general point, we should not assume sudo is available and configured in a way we can use it - the PMDA has root privileges by default so this is not an issue.